ui.R

```diff
@@ -91,7 +91,8 @@ ui <- shinydashboardPlus::dashboardPage(
   dashboardBody(
     tags$head(
       tags$style(sass(sass_file("www/scss/main.scss"))),
-      singleton(includeScript("www/js/readCookie.js"))
+      singleton(includeScript("www/js/readCookie.js")),
+      tags$script(htmlwidgets::JS("setTimeout(function(){history.pushState({}, 'Page Title', window.location.pathname);},2000);"))
```

Comment:
I [used](https://github.com/imCORE-DCC/data_curator/blob/production/ui.R#L94) "Data Curator" as the page title. 

```suggestion
      tags$script(htmlwidgets::JS("setTimeout(function(){history.pushState({}, 'Data Curator', window.location.pathname);},2000);"))
```